StRoot/StRefMultCorr/BadRun.h

```diff
@@ -66,16 +69,47 @@ const Int_t badrun_grefmult_2016[nBadRun_grefmult_2016] = {
 // 1. Au+Au 19.6 GeV (374)
 // 2. Au+Au 14.6 GeV (bad runs: 153 + injection runs: 403 = 556)
 // 3. Au+Au 200 GeV (5)
+// 4. Au+Au 4.59 AGeV (10) = 3.2 GeV
+// 5. Au+Au 7.3 AGeV (1) = 3.9 GeV
 //
-const Int_t nBadRun_refmult_2019 = 935;
+const Int_t nBadRun_refmult_2019 = 946;
```

Comment:
Add bad runs from Au+Au 7.3 AGeV = 3.9 GeV 
```suggestion
const Int_t nBadRun_refmult_2019 = 950;
```